### PR TITLE
feat: [C4R-937] landing page counter FE

### PR DESCRIPTION
**Pull request by OlteanuCosmin**
[C4R-937](https://wearetribus.atlassian.net/browse/C4R-937)

### CODE REVIEW

File: src/common/constants/CounterApp.constants.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export enum CounterApp {
+  PRACTICE_PROGRAM = 'practice-program',
+  CIVIC_SERVICE = 'civic-service',
```
Comment (dragos1195) [inline src/common/constants/CounterApp.constants.ts:3]: Why here?

File: src/pages/landing/components/counter/Counter.tsx [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
-  value: string;
+  value: number | undefined;
   type: COUNTER_TYPE;
 }
 
-interface CounterProps {
-  counterItems: ICounterItem[];
-}
+const Counter = () => {
+  const { data } = useGetLandingCountersQuery('civic-service');
```
Comment (dragos1195) [inline src/pages/landing/components/counter/Counter.tsx:12]: P4G, not CC

Comment (dragos1195) [reply src/pages/landing/components/counter/Counter.tsx:12]: Use ENUM not string

**Review by dragos1195 — CHANGES_REQUESTED** [review]

File: src/common/constants/CounterApp.constants.ts (hunk 1 above)
Comment (OlteanuCosmin) [reply src/common/constants/CounterApp.constants.ts]: Stersa valoarea.

File: src/pages/landing/components/counter/Counter.tsx (hunk 2 above)
Comment (OlteanuCosmin) [reply src/pages/landing/components/counter/Counter.tsx]: Used enum.

**Pull request merged**